habitat/plan.sh

```diff
@@ -69,9 +69,6 @@ wrap_inspec_bin() {
 #!$(pkg_path_for core/bash)/bin/bash
 set -e
 
-# Set binary path that allows InSpec to use non-Hab pkg binaries
-export PATH="/sbin:/usr/sbin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:$PATH"
```

Comment:
Another point of interest is that this `$PATH` will actually be evaluated at build-time and will include the build desp rather than just the runtime deps.

Reply:
I would have assumed this only affected build-time but I tested building an artifact with and without this line and it does change the `$PATH` when I'm running `hab pkg exec`

Reply:
Yes, sorry, what I mean is that `PATH` here is interpreted at build-time not run time. which is why you are getting all of the Hab binaries in your path.

Reply:
As in, this script is called at runtime, but the script that is rendered doesn't contain `$PATH` it contains the build-time expansion of that environment variable.

Reply:
_puts on glasses_

Oh this is rendering the bin script for InSpec. And `$PATH` gets expanded at build time. So this was extra buggy.

Reply:
@stevendanna reviewing this with @TheLunaticScripter , we currently need to put native bin dirs first. But we need to fix the $PATH containing build deps. I changed the commit to simply escape the $PATH variable at build time so it expands at runtime.